### PR TITLE
Release note for 1.8.0

### PR DESCRIPTION
**Pull request by ikhoon**

### CODE REVIEW

**Comment by ikhoon** [comment]
image

File: site/src/pages/release-notes/1.8.0.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+date: 2021-05-14
+---
+
+## 🌟 New features
+
+- You can now create a <type://HealthChecker> that invokes a health checker repetitively. #3392 #3406
```
Comment (minwoox) [inline site/src/pages/release-notes/1.8.0.mdx:7]: How about `... that updates its healthiness periodically.`?

File: site/src/pages/release-notes/1.8.0.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  client.prepare()
+        .maxResponseLength(maxResponseLength)
+        ...
+        .execute();
+  ```
+- You can now add or set HTTP headers and query params with `java.util.Map`. #3499 #3502
+  ```java
+   HttpHeaders
+      .builder()
+      .add(Map.of("foo", "1"));
```
Comment (minwoox) [inline site/src/pages/release-notes/1.8.0.mdx:41]:
```suggestion
      .add(Map.of("foo", "1"))
```

File: site/src/pages/release-notes/1.8.0.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+      .set(Map.of("bar", "2"));
+  
+  QueryParams.builder()
+             .add(Map.of("page", "1"));
```
Comment (minwoox) [inline site/src/pages/release-notes/1.8.0.mdx:45]:
```suggestion
             .add(Map.of("page", "1"))
```

File: site/src/pages/release-notes/1.8.0.mdx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+             .set(Map.of("size", "2"));
+  ```
+- You can now disable DNS query metrics using <type://DnsResolverGroupBuilder#disableDnsQueryMetrics()>.
+  #3368 #3552
+  ```java
+  ClientFactory.builder()
+             .domainNameResolverCustomizer(customizer -> {
```
Comment (minwoox) [inline site/src/pages/release-notes/1.8.0.mdx:52]:
```suggestion
  ClientFactory.builder()
               .domainNameResolverCustomizer(customizer -> {
```

File: site/src/pages/release-notes/1.8.0.mdx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+## 🌟 New features
+
+- You can now create a <type://HealthChecker> that invokes a health checker repetitively. #3392 #3406
+  ```java
+  HealthChecker.of(() -> {
+      HealthCheckStatus status = ...;
+      CompletableFuture.completedFuture(status);
+  }, Duration.ofSeconds(5));
+  ```
+- You can now create a propagating <type://ContextAwareExecutorService> and
```
Comment (minwoox) [inline site/src/pages/release-notes/1.8.0.mdx:14]:
```suggestion
- You can now create the <type://RequestContext> propagating <type://ContextAwareExecutorService> and
```

**Review by minwoox — APPROVED** [review]
Left some nits. Thanks!

**Review by trustin — APPROVED** [review]

File: site/src/pages/release-notes/1.8.0.mdx (hunk 5 above)
Comment (ikhoon) [reply site/src/pages/release-notes/1.8.0.mdx:14]: `RequestContext.makeContextPropagating()` actually returns a `ExecutorService` or `ScheduledExecutorService`

**Pull request merged**